src/modules.py

```diff
@@ -162,9 +162,20 @@ def forward(self, sent, task, reset=True):
             sent_embs = self._dropout(sent_embs)
         if task_sent_embs is not None:
             task_sent_embs = self._dropout(task_sent_embs)
+        max_seq_length = (sent_embs.shape[1]
+                          if sent_embs is not None
+                          else task_sent_embs.shape[1])
 
         # The rest of the model
         sent_mask = util.get_text_field_mask(sent).float()
+        if sent_mask.shape[1] < max_seq_length:
```

Comment:
I'm not sure this will ever trigger since `sent_mask` is the same shape as `sent_embs` so its `shape[1]` will be the same as `sent_embs.shape[1]`.